Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/FormsTokenizationViewModel/Fields/AddressField.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  LastNameField.swift
+//  PrimerSDK
+//
+//  Created by Dario Carlomagno on 08/06/22.
+//
+
+#if canImport(UIKit)
+
+import UIKit
+
+class PrimerAddressField: PrimerCardFormFieldProtocol {
+    
+    internal static let theme: PrimerThemeProtocol = DependencyContainer.resolve()
+}
+
+class PrimerAddressLine1Field: PrimerAddressField {
+        
+    static func addressLine1ContainerViewFieldView(_ view: PrimerTextFieldView) -> PrimerCustomFieldView {
+        let addressLine1FieldContainerView = PrimerCustomFieldView()
+        addressLine1FieldContainerView.fieldView = view
+        addressLine1FieldContainerView.placeholderText = Strings.CardFormView.AddressLine1.label
+        addressLine1FieldContainerView.setup()
+        addressLine1FieldContainerView.tintColor = theme.input.border.color(for: .selected)
+        return addressLine1FieldContainerView
+    }
+    
+    static func addressLine1FieldViewWithDelegate(_ delegate: PrimerTextFieldViewDelegate?) -> PrimerAddressLine1FieldView {
+        let addressLine1FieldView = PrimerAddressLine1FieldView()
+        addressLine1FieldView.placeholder = Strings.CardFormView.AddressLine1.placeholder
+        addressLine1FieldView.heightAnchor.constraint(equalToConstant: 36).isActive = true
+        addressLine1FieldView.textColor = theme.input.text.color
+        addressLine1FieldView.delegate = delegate
+        return addressLine1FieldView
+    }
+}
+
+class PrimerAddressLine2Field: PrimerAddressField {
```

Comment:
Is there any difference between `PrimerAddressLine1Field` and `PrimerAddressLine2Field`? Can't the `PrimerAddressLine1Field` be renamed to `PrimerAddressLineField` and be reused?

Reply:
`PrimerAddressLine2Field` contains a different validation, and other analytics fields etc...other than that nothing much.
Though I wanted to make it two different fields as we make a "class compare" in `CardFormPaymentMethodTokenizationViewModel.swift:869` so I didn't want to disrupt the logic